ansible/roles/test/tasks/decap.yml

```diff
@@ -19,6 +19,14 @@
        - sonic_hwsku in mellanox_hwskus
        - dscp_mode is not defined
 
+  - name: Set dscp_mode for decap test for marvell
+    set_fact:
+       dscp_mode: pipe
+       ecn_mode: copy_from_outer
+    when:
+       - sonic_hwsku in marvell_hwskus
```

Comment:
can you add this as an "or" check to line 11 instead of defining the same thing again. You can update the comment.

Reply:
Hi prsunny

   I have modified the changes according to review comment, Could you please review this

Thanks
Rajneesh

Reply:
can you remove this part?

Reply:
I have already removed these lines
is it still showing ?

Thanks
Rajneesh

Reply:
https://github.com/Azure/sonic-mgmt/pull/1228/commits/ef3c1eda93de845e20f733feed77ca908cd8cf2d